src/pupil_labs/invisible_lsl_relay/xdf_cloud_time_sync.py

```diff
@@ -1,70 +1,78 @@
 import json
-import os.path
+import logging
 import pathlib
 
 import click
 import pyxdf
 
 
+logger = logging.getLogger(__name__)
+
+
 @click.command()
-@click.argument('path_to_xdf', nargs=1)
-@click.argument('paths_to_exports', nargs=-1)
+@click.argument('path_to_xdf', nargs=1, type=click.Path(exists=True))
+@click.argument('paths_to_exports', nargs=-1, type=click.Path(exists=True))
```

Comment:
One can even go a step further and tell click if one is expecting directories or files
```suggestion
@click.argument('path_to_xdf', nargs=1, type=click.Path(exists=True), file_okay=True, dir_okay=False)
@click.argument('paths_to_exports', nargs=-1, type=click.Path(exists=True), file_okay=False, dir_okay=True)
```